tests/phpunit/Command/GenerateProfilesAndSourcesCommandTest.php

```diff
@@ -73,7 +74,9 @@ public function testGenerateProfilesAndSourcesCommand(
         $this->assertEquals(Command::SUCCESS, $exitCode, $output);
         $this->assertStringContainsString('Sources and profiles.yml files generated.', $output);
 
-        [$workspace] = $this->workspaceManagementService->getConfigurationWorkspaces(self::KBC_DEV_TEST);
+        $workspaces = $this->workspaceManagementService->getConfigurationWorkspaces(self::KBC_DEV_TEST);
+        $workspace = (array) array_pop($workspaces);
+
```

Comment:
Jen drobne ladeni, aby ty testy byli stabilnejsi

Reply:
Takze tady vyberu posledni workspace (ne prvni). I kdyz by tam mel byt jenom jeden jak se to ted maze na zacatku.